pkg/descheduler/evictions/evictions.go

```diff
@@ -99,6 +99,7 @@ func (pe *PodEvictor) TotalEvicted() int {
 	for _, count := range pe.nodepodCount {
 		total += count
 	}
+	metrics.TotalPodsEvicted.With(map[string]string{"result": "total pods evicted so far"}).Inc()
```

Comment:
The reason for incrementing the metrics here is to capture the total pods evicted as per the count goes up in line 100. I thought of keeping it align with the evicted pod count going up in `evictions.go` so that it might be easier to record the total count. Let me know your thoughts otherwise I'll make some changes.

Reply:
I see. It seems odd that it's being incremented in a method that is only supposed to get total evicted count.
```
// TotalEvicted gives a number of pods evicted through all nodes
```

Reply:
Updated this as well to [this](https://github.com/kubernetes-sigs/descheduler/pull/648/files#diff-695c3df113e19fd5eaebf2b4bd6f6d4ba498c1bd0ae97875816360b66f6f8044R133) line.